sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -333,6 +333,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val PLAN_EXPLAIN_EXTENSIONS_ENABLED = buildConf("spark.sql.plan.explain.extensions.enabled")
```

Comment:
This introduces 3 new namespaces in Apache spark configurations. We should avoid this as much as possible.
```
spark.sql.plan.*
spark.sql.plan.explain.*
spark.sql.plan.explain.extensions.*
```

Can we change like `spark.sql.planChangeValidation`?